src/porepy/geometry/geometry_property_checks.py

```diff
@@ -1,5 +1,5 @@
 """
```

Comment:
You need to add a line here:
`from __future__ import annotations`
So the usage of, for example, `list` instead of  `List` will work with python3.8.